test/DoubleConstantsTest.thrift

```diff
@@ -0,0 +1,17 @@
+namespace java thrift.test
+namespace cpp thrift.test
+
+// more tests on double constants (precision and type checks)
```

Comment:
Further evidence that disabling the test in Java is a bad idea: why don't you need to disable Javascript if the compiler is the cause?

Reply:
If I'm not mistaken, only Java and Python tests are running on Windows builders. One of the C++ compilers on these builders compiles the Thrift compiler source code such that the produced relatively large double constants does not match the ones given in the test. That's why I have disabled `TestRenderedDoubleConstants` only for Java and Python.

Reply:
There's probably another way to resolve this, such as fixing a compiler flag being passed to older MSVC2013 compilers.  I'll swing back around to this one when I can.  I don't like the notion that we say, "because the compiler was built with MSVC2013, let's disable a test".  Shouldn't any thrift compiler, regardless of the C++ compiler used to create it, produce working thrift generated code that has the same values as any other thrift compiler?